### PR TITLE
Fixed GitAddCommand

### PR DESCRIPTION
**Pull request by matweew**
README file updated (warning message)

Fixed `git add` command params to do not add new files (`git add --update`)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,9 +62,13 @@ preCommitFileCommands:
     - GitHook\Command\FileCommand\PreCommit\PhpStanCheckCommand
 
 preCommitRepositoryCommands:
+    - GitHook\Command\RepositoryCommand\PreCommit\ValidateBranchNameCommand
     - GitHook\Command\RepositoryCommand\PreCommit\GitAddCommand
 ```
 
+See the full list of available commands in the `src/GitHook/Command/FileCommand` and `src/GitHook/Command/RepositoryCommand`.
```
Comment (dereuromark) [inline README.md]:
```suggestion
See the full list of available commands in `src/GitHook/Command/FileCommand/` and `src/GitHook/Command/RepositoryCommand/`.
```

**Comment by dereuromark** [comment]
Did you also remove the default include in code?

**Comment by matweew** [comment]
it should be done in suite-nonsplit. still under discussion

**Comment by matweew** [comment]
Fix this fix it also good to add 

`CodeStyleFixCommand` instead of `CodeStyleFixCommand` in suite .githook! it works like a charm

**Review by dereuromark — APPROVED** [review]

**Pull request merged**